packages/trigger-sdk/package.json

```diff
@@ -56,6 +54,6 @@
     "typescript": "^4.8.4"
   },
   "engines": {
-    "node": ">=16.8.0"
+    "node": "^18.0.0"
```

Comment:
This limits the Node version to >= 18 and < 19, which isn't what we want. I'd just do `>=18.0.0`